src/components/map/MapApi.js

```diff
@@ -1,5 +1,8 @@
-import MapApi, { layerTypes, controlTypes } from '@dhis2/gis-api';
-// import MapApi, { layerTypes, controlTypes } from '@dhis2/maps-gl';
+import MapApi, {
+    layerTypes,
+    controlTypes,
+    loadEarthEngineApi,
```

Comment:
Will this work with `maps-gl` as well?

Reply:
Yes it will. maps-gl will not have earth engine bundled, while this is still the case for gis-api.